cmd/root.go

```diff
@@ -97,7 +150,7 @@ func GetJIRAClient(username, password, baseURL string) (*jira.Client, error) {
 		log.Errorf("Error connecting to JIRA; check your credentials. Error: %s", jira.CheckResponse(resp.Response))
 		return nil, jira.CheckResponse(resp.Response)
 	}
-	project = *proj
+	project = *(proj.(*jira.Project))
```

Comment:
this is now a pointer to a pointer. is that what you want? also, make sure to check if the type assertion succeeds.